src/platform_impl/windows/window_state.rs

```diff
@@ -48,6 +48,7 @@ pub struct MouseProperties {
     pub cursor: CursorIcon,
     pub buttons_down: u32,
     cursor_flags: CursorFlags,
+    pub last_position: Option<(f64, f64)>,
```

Comment:
Should be `PhysicalPosition` (or `LogicalPosition`, but I think most likely the former).

Reply:
You mean Option<PhysicalPosition<P>> ?
I considered it but wouldn't that cause the P to trickle up MouseProperties, WindowState, etc ?

Reply:
I went for `Option<PhysicalPosition<f64>>`